src/app/pages/Kafka/metricsApi.ts

```diff
@@ -0,0 +1,195 @@
+import {
```

Comment:
I do wonder if this should be pushed up into the SDK.

Reply:
@wtrocki WDYT?

Reply:
+100 for including any service level. I'm happy to facilitate that unless you feel that this will need to evolve fast and having it in the separate project that requires release will impede your speed. 

Maybe it will be better to address that on complex level - creating micro epic and take over service abstractions for every service

Reply:
These functions are built to serve the metrics dashboard. I'm not sure having these functions in a generic purpose SDK would be helpful, but I'd argue that a way to get the metrics out of the system without having to know exactly how they are stored in the backend would be nice.